support-frontend/test/utils/CheckoutValidationRulesTest.scala

```diff
@@ -370,6 +370,24 @@ class SimpleCheckoutFormValidationTest extends AnyFlatSpec with Matchers {
     SimpleCheckoutFormValidation.passes(requestWithLongLastName) shouldBe an[Invalid]
   }
 
+  it should "reject invalid characters in strings" in {
+    SimpleCheckoutFormValidation.noFourByteUtf8Characters("test1", "hello😊") shouldBe an[Invalid];
+    SimpleCheckoutFormValidation.noFourByteUtf8Characters("test2", "𐀀goodbye") shouldBe an[Invalid];
+    SimpleCheckoutFormValidation.noFourByteUtf8Characters("test3", "wa𠀀it") shouldBe an[Invalid];
+  }
+
+  it should "not reject valid characters in strings" in {
+    SimpleCheckoutFormValidation.noFourByteUtf8Characters("test1", "hello✅") shouldBe Valid;
```

Comment:
We were preventing this checkmark from being sent before (as it’s an emoji), but Zuora can actually handle it fine, as it’s under U+FFFF.